src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10006,7 +10003,7 @@ private TupleExpressionSyntax ParseTupleExpressionTail(SyntaxToken openParen, Ar
             }
         }
 
-        private bool ScanCast()
+        private bool ScanCast(bool forPattern = false)
```

Comment:
>bool forPattern = false [](start = 30, length = 23)

The parameter is only passed to ScanType and it is not used there.

Reply:
Recorded in https://github.com/dotnet/roslyn/issues/23222.